### PR TITLE
D8CORE-4194 Use a regular link markup and stretch it over the image

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Instead of wrapping the image in a link, display the link separately and stretch it over the image. This way the screen reader will read them separately instead of a single element without a title on the link tag.

# Need Review By (Date)
- 6/3

# Urgency
- medium

# Steps to Test
- follow steps in https://github.com/SU-SWS/stanford_profile/pull/423

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: templates/components/media/media.html.twig [hunk 1]
```diff
@@ -18,4 +16,121 @@
 {% set modifier_class = attributes.class %}
 {% set attributes = attributes|without('class') %}
 
-{% extends "@decanter/components/media/media.twig" %}
```
Comment (sherakama) [inline templates/components/media/media.html.twig:21]: This seems like a useful change to the decanter package. Why punt?

Comment (pookmish) [reply templates/components/media/media.html.twig:21]: shrug

**Comment by cjwest** [comment]
@pookmish, @ishahism, and @sherakama ,
I want to start by saying, I like how this functions with VoiceOver. 

I presented this approach to folks at SODA to get their feedback and insight. 

1. Seems over engineered, difficult to maintain, 
2. We would need to test this on mobile devices and with JAWS and NVDA.

The big issue is still that the sighted user doesn't know 1) that the image is a link and 2) what will happen when you click the link. 
SODA folks agreed and added that a Dragon user (voice only navigation) wouldn’t know either. 

I think we need to fix the UX for this  and some how display the link text to the user before we code up a solution.

**Comment by pookmish** [comment]
1. it's not reallly over engineered or difficult to maintain. it was less than 10 lines of css to accomplish this. We determine our own maintenance, not SODA.
2. A mobile device should be no different than a desktop.

But yes the design should probably change to make it better for sighted users.

**Comment by cjwest** [comment]
Sanity check, @pookmish - I see that this is marked "ready for review". Is that accurate?

**Comment by pookmish** [comment]
No changes have been made since your original review so if there are changes necessary, then they haven't been completed.

**Comment by cjwest** [comment]
@pookmish,
I want to know when to review this ticket.
Do you want to mark this as "Not ready for review" or should I take my name off?

**Comment by cjwest** [comment]
Please assign this to me when it's ready for my review

**Review by cjwest — APPROVED** [review]

**Comment by cjwest** [comment]
Thanks Mike!

**Pull request merged**